src/places/pages/Places.vue

```diff
@@ -1,22 +1,401 @@
+                </strong>
+              </QChip>
+            </RouterLink>
+            <QIcon
+              v-if="place.isSubscribed"
+              class="q-ml-sm self-center"
+              name="fas fa-star"
+              color="secondary"
+              :title="$t('PLACE_LIST.SUBSCRIBED')"
+            />
+          </div>
+
+          <div class="q-ml-md q-mr-xs q-mt-xs limit-height text-grey-9">
+            <Markdown
+              v-if="place.description"
+              :source="place.description"
+            />
+            <div v-else>
+              <span class="text-italic">
+                {{ $t("STOREDETAIL.NO_DESCRIPTION") }}
+              </span>
+            </div>
+          </div>
+        </QCard>
+      </div>
+    </div>
+
+    <div
+      v-if="hasNoPlacesDueToFilters"
+      class="q-pa-md"
+    >
+      <QBanner
+        class="bg-white"
+        :inline-actions="$q.platform.is.desktop"
+      >
+        <template #avatar>
+          <QIcon
+            name="fas fa-info-circle"
+            color="grey"
+          />
+        </template>
+        <h5 class="q-ma-xs">
+          {{ $t('PLACE_LIST.NONE_DUE_TO_FILTER') }}
+        </h5>
+        <template #action>
+          <QBtn
+            flat
+            @click="() => showAll()"
+          >
+            {{ $t('STOREEDIT.SHOW_ALL') }}
+          </QBtn>
+        </template>
+      </QBanner>
+    </div>
+  </div>
 </template>
 
 <script setup>
-import { QCard } from 'quasar'
+import {
+  QCard,
+  QSelect,
+  QBtn,
+  QIcon,
+  QItem,
+  QItemSection,
+  QItemLabel,
+  QBanner,
+  QCheckbox,
+  QInput,
+  QChip,
+  debounce,
+} from 'quasar'
+import { computed, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
 
+import { useActivityListQuery } from '@/activities/queries'
 import { useCurrentGroupService } from '@/group/services'
+import { statusList } from '@/places/placeStatus'
+import { placeRoute } from '@/places/utils'
+import { useStatusService } from '@/status/services'
+import { useQueryParams } from '@/utils/mixins/bindRoute'
+import { newDateRoundedTo5Minutes } from '@/utils/queryHelpers'
+
+import Markdown from '@/utils/components/Markdown.vue'
 
-import PlaceList from '@/places/components/PlaceList'
+import { usePlaceHelpers, usePlaceTypeHelpers } from '../helpers'
+import { usePlaceTypeService } from '../services'
 
 const {
+  groupId,
   places,
-  archivedPlaces,
+  isEditor,
 } = useCurrentGroupService()
+
+const {
+  getPlaceStatus,
+} = useStatusService()
+
+const {
+  getPlaceIconProps,
+} = usePlaceHelpers()
+
+const {
+  getTranslatedName,
+  getIconProps,
+  sortByTranslatedName,
+} = usePlaceTypeHelpers()
+
+const {
+  getPlaceTypesByGroup,
+  getPlaceTypeById,
+} = usePlaceTypeService()
+
+const {
+  activities,
+  isFetching: isFetchingActivities,
+  hasNextPage,
+  fetchNextPage,
+} = useActivityListQuery({
+  groupId,
+  dateMin: newDateRoundedTo5Minutes(),
+  pageSize: 1200,
```

Comment:
As I said elsewhere, I _think_ because this isn't part of the cache key, other calls to `useActivityListQuery` might fight with it and cause the number flickering that I've seen.

One solution could be to create a separate query for the purpose of getting the counts, e.g. `useActivityCountQuery` .. that does the fetching ALL pages + creating the counts logic?